### PR TITLE
Fix 575483 - update user async should do full replace

### PR DESCRIPTION
**Pull request by laura-rodriguez**

### CODE REVIEW

File: openapi3/management.yaml [hunk 1]
```diff
@@ -13930,7 +13930,7 @@ paths:
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/UpdateUserRequest'
+              $ref: '#/components/schemas/User'
```
Comment (bryanapellanes-okta) [inline openapi3/management.yaml:13933]: Will this require broader review, or is this somehow specific to just the .NET Sdk? Or, has it already been reviewed?

**Review by bryanapellanes-okta — COMMENTED** [review]
Will this require a version update?

**Comment by laura-rodriguez** [comment]
> Will this set of changes require a version update?

I kept the previous interface and marked it as obsolete to avoid breaking changes.

**Review by bryanapellanes-okta — APPROVED** [review]
👍

**Pull request merged**